account_invoice_crm_tag/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module allows you to add crm tags to invoices coming from sales orders
```

Comment:
You can put manually the tags. The propagation from sales order is an extra feature that you should mention also, but not only.